### PR TITLE
Feature/labs38 notebook final 

### PR DESCRIPTION
**Pull request by gaben3722**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

The model was tested in the performance notebook displaying various metrics. 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
https://www.loom.com/share/82645549702849ab84cc5de22d6c55ac

### CODE REVIEW

**Review by ChristopherKchilton — APPROVED** [review]
Based on the video it appears the notebook was changed with some improvements. I approve of the changes done in this pull request.

**Review by iknight7000 — APPROVED** [review]
The video definitely helped with the explanation. I like the new improvements. There is a big difference between a 1 and 5.

**Review by mikecarrier4 — APPROVED** [review]
more concise.  Video explains functionality for future cohorts . Good Job

File: notebooks/labs38_notebooks/FrankenBERT_Performance.ipynb [hunk 1]
```diff
@@ -56,7 +56,8 @@
    ],
    "source": [
     "!pip install transformers\n",
-    "!pip install psycopg2-binary"
+    "!pip install psycopg2-binary\n",
```
Comment (iknight7000) [inline notebooks/labs38_notebooks/FrankenBERT_Performance.ipynb:59]: This is the commit detail I mentioned.

**Comment by BrokenShell** [comment]
this was carried over to another PR

**Pull request closed without merging**